### DISCUSSION TITLE
Is scalar index (e.g. Inverted Index) loaded in memory?

### DISCUSSION THREAD

**Discussion by pycui** [milvus-io/milvus#35906] [Q&A and General discussion] [answered]
When calculating the memory requirement of query nodes, how to properly consider the scalar indexes? Can we add more documentation on this? thanks

**Comment by yhmo** [comment, marked as answer]
Scalar index is loaded into memory.
The document already shows how to estimate scalar index size: https://milvus.io/docs/scalar_index.md

**Comment by xiaofan-luan** [comment]
with latest 2.4.10, you can enable mmap for  some of the inverted index.

@SimFG 
do we have a document for that?
